<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterDialog</class>
 <widget class="QDialog" name="ParameterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="gaussBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>高斯半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>σ：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sigmaEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>00.00</string>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
  <widget class="QLabel" name="tLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>阈值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
   <property name="placeholderText">
    <string>0-255</string>
   </property>
  </widget>
  <widget class="QLabel" name="ltLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>低阈值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="htLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>高阈值：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ltEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000</string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
   <property name="placeholderText">
    <string>0-255</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="htEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
   <property name="placeholderText">
    <string>0-255</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="houghTEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>0.00</string>
   </property>
   <property name="text">
    <string>0.5</string>
   </property>
  </widget>
  <widget class="QComboBox" name="angleBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>500</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="houghTLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>霍夫阈值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="angleLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>角度等分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="houghTiLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="angleiLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>π/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>390</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>高斯平滑滤波器设定：</string>
   </property>
  </widget>
  <widget class="QLabel" name="tresholdlabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>边缘检测阈值设定：</string>
   </property>
  </widget>
  <widget class="QLabel" name="houghLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>霍夫变换直线相关设定：</string>
   </property>
  </widget>
  <widget class="QFrame" name="line_0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>110</y>
     <width>341</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="line_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>341</width>
     <height>1</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::HLine</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QLabel" name="circlelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>霍夫变换圆相关设定：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rMaxLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>最大半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="rMinLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>最小半径：</string>
   </property>
  </widget>
  <widget class="QLabel" name="radiusLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>半径等分：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rMaxEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rMinEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>000</string>
   </property>
  </widget>
  <widget class="QComboBox" name="radiusBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
